src/main/java/adris/altoclef/butler/ButlerConfig.java

```diff
@@ -51,4 +51,19 @@ public static ButlerConfig getInstance() {
      */
     public boolean whisperFormatDebug = false;
 
+    /**
+     * If set to true, will reply with "Sorry, you're not authorized!".
+     *
+     * Disable this if you need to stay undercover.
+     */
+    public boolean sendAuthorizationResponse = true;
+
+    /**
+     * Use this to customize the response if "sendAuthorizationResponse"
+     * 
+     * You may use {from} to get sender username.
+     * 
+     * Disable this if you need to stay undercover.
+     */
```

Comment:
replace javadoc with

/**
\* The response sent in a failed execution due to non-authorization
\* @param from: the username of the player who triggered the failed authorization response
*/